driver-core/src/main/com/mongodb/connection/SimpleCommandProtocol.java

```diff
@@ -30,7 +30,7 @@
 import static java.lang.String.format;
 
 /**
- * A protocol for executing a command against a MongoDB server using the OP_QUERY wire protocol message.
```

Comment:
As with SimpleCommandMessage, this class can be merged with the splittable payload variant into a single class with an optional payload.

Reply:
Done.